docs/resources/anycast_config.md

```diff
@@ -0,0 +1,86 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_anycast_config Resource - terraform-provider-bloxone"
+subcategory: ""
```

Comment:
The subcategory should be added so that the documentation is grouped in the sidebar. 

To add to this auto generated docs, you will have to change these template to add "anycast" as a group.

https://github.com/infobloxopen/terraform-provider-bloxone/blob/master/templates/resources.md.tmpl#L4-L15

https://github.com/infobloxopen/terraform-provider-bloxone/blob/master/templates/data-sources.md.tmpl#L4-L15

Reply:
added